packages/components/pie-checkbox/src/index.ts

```diff
@@ -164,13 +165,26 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
 
         const componentDisabled = disabled || disabledByParent;
 
+        const checkboxClasses = {
+            'c-checkbox': true,
+            [`c-checkbox--status-${status}`]: !componentDisabled && status,
+            'c-checkbox--disabled': componentDisabled,
+            'c-checkbox--checked': checked,
+            'c-checkbox--indeterminate': indeterminate && !checked,
+        };
+
+        const labelClasses = {
+            'c-checkbox-tick': true,
+            [`c-checkbox-tick--status-${status}`]: !componentDisabled && status,
```

Comment:
`status` will always be truthy, I believe?
```suggestion
            [`c-checkbox-tick--status-${status}`]: !componentDisabled,
```